src/scenes/UserGalleryPage/QRCode.tsx

```diff
@@ -0,0 +1,149 @@
+        dotsOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#6a1a4c',
+            color2: '#6a1a4c',
+            rotation: '0',
+          },
+        },
+        cornersSquareOptions: { type: '', color: '#000000' },
+        cornersSquareOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#000000',
+            color2: '#000000',
+            rotation: '0',
+          },
+        },
+        cornersDotOptions: { type: '', color: '#000000' },
+        cornersDotOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#000000',
+            color2: '#000000',
+            rotation: '0',
+          },
+        },
+        backgroundOptionsHelper: {
+          colorType: { single: true, gradient: false },
+          gradient: {
+            linear: true,
+            radial: false,
+            color1: '#ffffff',
+            color2: '#ffffff',
+            rotation: '0',
+          },
+        },
+      });
+
+      qrCode.append(ref.current);
+    }
+  }, [ref, username]);
```

Comment:
don't think we need `ref` in the dependency array here! unless eslint is asking for it?